src/style.css

```
@@ -0,0 +1,87 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0 10px 50px;
+  display: flex;
+}
+
+/*--------------------------------------*/
+
+.link__item {
+  margin-left: 20px;
+  padding: 0;
+  list-style: none;
+  box-sizing: border-box;
+  display: flex;
+}
+
+.link {
+  height: 60px;
+  line-height: 60px;
+  margin: 0;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-family: inherit;
+  font-size: inherit;
+  display: flex;
+  box-sizing: border-box;
+  justify-content: space-between;
+  color: inherit;
+}
+
+.nav {
+  height: 60px;
+  margin: 0 50px 0 0;
+  box-sizing: border-box;
+}
+
+.nav__ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.header {
+  height: 60px;
+  margin-top: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  height: 4px;
+  width: 37px;
```

```suggestion
  width: 100%;
```
або `display: block`